### DISCUSSION TITLE
Error: Maximum update depth exceeded

### DISCUSSION THREAD

**Discussion by DroopeGu1** [KevinVandy/material-react-table#921] [Q&A]
I'm doing a project with next.js and using the table with Column Ordering with Drag and Drop, but I'm getting this error, I used the example from the documentation. 
I get this error when using the show filters toggle

![image](https://github.com/KevinVandy/material-react-table/assets/102196495/a3c90076-e824-4f43-9b89-21db487b96b0)

**Comment by KevinVandy** [comment]
I see you closed this. Was it because data did not have a stable reference?
